features/google/src/context/google-context.jsx

```diff
@@ -14,18 +16,30 @@ function updatePodNavigation(pod, history, handleBack, location) {
         : pod.polyNav.setActiveActions([]);
 }
 
+function updateTitle(pod, location, popUp) {
+    console.log(location.pathname.substring(1));
+    console.log(popUp);
+    pod.polyNav.setTitle(
+        location.pathname === "/"
+            ? ""
+            : popUp
+            ? i18n.t("navbarTitles:info")
```

Comment:
That's neat, but I don't think we're supposed to change the title just because a popup is shown. In _facebookImport_ we have the HTRT screens as actual screens, not just popups, so we needed some dedicated title there.

Reply:
Do you think that's in scope? We basically tried to make the titles change when they had to.

Reply:
Well, my point is that setting a different title when a popup is showing up is not in scope, not the other way around :D Wouldn't kill me to merge it like this, but I just don't think it's the desired behaviour - plus it's extra logic.